### DISCUSSION TITLE
Signature dictionary entries support

### DISCUSSION THREAD

**Discussion by eduperottoni** [MatthiasValvekens/pyHanko#307] [General]
Hey, @MatthiasValvekens!

According to ISO3200-2 (2020), there are many entries in the PDF's signature dictionary (table 255), e.g. `/R`, `/M`, `/Name`, `/Reason`, `/Location`, `/Prop_Build`, `/Prop_AuthTime`, `/Prop_AuthType`, `/ByteRange` and others. On the other hand, in pyHanko's API, we only have some of them exposed in the `SignatureObject`, being them: `timestamp,  subfilter, name, location, reason, app_build_props and bytes_reserved`. 
I know it's easy to inherit this class and create new entries to be part of my signature dictionary (and I did for `/Prop_AuthTime` and `/ContactInfo` entries, for example). However, is there any reason to pyHanko's lack of support to this entries?

Thank you!

**Comment by MatthiasValvekens** [comment]
Almost all of these are supported, actually (see e.g. #195 for the PR that adds `/Prop_Build` support).

The ones that aren't supported (e.g. `/R` and some of the `/Prop_XYZ` ones) aren't included because nobody asked so far :). Feel free to submit a PR.

**Reply by eduperottoni** [reply]
Oh, I forgot to notify before to link here, but the PR #314 was done in order to solve this issue.
